src/python/pants/option/global_options.py

```diff
@@ -111,6 +115,10 @@ def from_bootstrap_options(cls, bootstrap_options):
     remote_execution_extra_platform_properties=[],
     remote_execution_headers={},
     process_execution_local_enable_nailgun=False,
+    process_execution_local_symlink_optimization_threshold=100,
+    # The default is set to clean up cached materializations if they have not been accessed for 2
+    # days.
+    process_execution_local_symlink_ttl=172800.0,
```

Comment:
It's a bit clearer if you write this as 60 * 60 * 24 * 2. It would be even better if there's some standard python library that would let you write something human-readable like "2 days" and convert it into seconds that way, but I don't know of one offhand.